plugins/pipelines/app/models/concerns/pipeline_plugin_concern.rb

```diff
@@ -0,0 +1,33 @@
+module PipelinePluginConcern
+  # Return true if any stages in the pipeline are marked production
+  def production?
+    super || Stage.find(next_stage_ids).any? { |s| s.production? }
```

Comment:
oh yes, must be some new 4.x thing :D

On Thu, Jul 23, 2015 at 8:58 AM, Shane Hender <notifications@github.com>
wrote:

> In plugins/pipelines/app/models/concerns/pipeline_plugin_concern.rb
> <https://github.com/zendesk/samson/pull/425#discussion_r35337836>:
>
> > @@ -0,0 +1,33 @@
> > +module PipelinePluginConcern
> > +  # Return true if any stages in the pipeline are marked production
> > +  def production?
> > +    super || Stage.find(next_stage_ids).any? { |s| s.production? }
>
> Yup, to the .any?(&:production?), but it also seems activerecord is smart
> enough to skip the find if passed an empty array.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/425/files#r35337836>.
>